client/src/App.js

```diff
@@ -106,35 +101,21 @@ class App extends Component {
     }
 
     sendMessage(text) {
-        //this.state.messages.push({
-        //    'text': text,
-        //    // name: this.state.email
-        //})
-        for (let message in tempMessages){
-            this.state.messages.push(tempMessages[message]);
+        const requestOptions = {
+            method: 'Post',
+            headers: {'Content-Type': 'application/json'},
+            body: JSON.stringify({"sender": localStorage.getItem('login'), "recipients": fetch(url+"users/"), "contents": text})
         }
+        /*fetch(url + "messages/", requestOptions)
```

Comment:
since we're switching from http to socket with this pr, just get rid of all the old stuff commented out that has to do with sending messages

Reply:
Should be all good now, went ahead and renamed the socket names as well.